stories/pagination.stories.js

```diff
@@ -14,25 +14,26 @@ export const Normal = () => {
   const [data, setData] = useState({
     exampleData: exampleData,
     pageOfItems: [],
+    pager: {},
   })
 
   const headerClass = 'bg-primary-200 text-black text-center h-full'
   const contentClass = 'bg-primary-400 text-black text-center h-full w-full'
 
-  const onChangePage = (pageOfItems) => {
-    setData({ ...data, pageOfItems: pageOfItems })
+  const onChangePage = (_, val) => {
+    setData({ ...data, pageOfItems: val.pageOfItems, pager: val._pager })
   }
 
   return (
     <Layout>
       <Layout.Header className={headerClass}>Normal type with default styling</Layout.Header>
       <Layout.Content className={contentClass}>
-        {data.pageOfItems.map(item =>
-          <div key={item.id}>{item.name}</div>,
-        )}
+        {data.pageOfItems.map((item, index) => {
+          return (<div key={index}>{data.exampleData[item.i].name}</div>)
```

Comment:
No need to add return here. Can simplify:
`
{data.pageOfItems.map((item, index) => (
  <div key={index}>{data.exampleData[item.i].name}</div>
))}
`